docs/platform/howto/pro-platform.rst

```diff
@@ -0,0 +1,36 @@
+Pro Platform
+=============
+
+Pro Platform gives enterprise organizations added capabilities for implementing security, resilience, reliability, and cost-efficiency strategies for their data infrastructure. Pro Platform features are available across disaster recovery, security and compliance, advanced networking, and user lifecycle management.
+
+To use Pro features, enable Pro Platform for your organization.  
```

Comment:
we've temporary removed the self-serve path, it can be only enabled/disabled via sales